Assets/Scripts/Blocks/BlockData/RespawnBlock_Data.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+[System.Serializable]
+public class RespawnBlock_Data : Block_Data
+{
+    public override BlockType BlockType => BlockType.Respawn;
+
+    private float respawnTime;
```

Comment:
Is it really necessary to save the spawnTime? it should be the the same through all the blocks, right?

Reply:
Ok

Reply:
No only the respawnBlock has a respawnTime, this time is the time that passes until the respawnBlock reapears.